src/web/recorder/webpack.config.js

```diff
@@ -19,7 +19,10 @@ module.exports = {
     rules: [
       {
         test: /\.(j|t)sx?$/,
-        use: 'ts-loader',
+        loader: 'ts-loader',
```

Comment:
Is this using babel? If so, which version?

Reply:
Replaced this with babel